src/main/java/serenitylabs/tutorials/vetclinic/collections/katas/PetHotel.java

```diff
@@ -17,9 +18,20 @@
         return new ArrayList<>(pets);
     }
 
-    private enum HotelAvailability { Available, Full}
+    public void feedTheGuests() {
+        for (Pet pet : getPets()) {
+            if (pet.getBreed() == Breed.Cat) {
+                pet.feed(10 * pet.getWeightInKilos(), PetFood.KittyKat);
+            } else if (pet.getBreed() == Breed.Dog) {
+                pet.feed(20 * pet.getWeightInKilos(), PetFood.FidosFood);
+            }
```

Comment:
This code contains a for look and a nested if. When this happens, it can add extra complexity to the code, and make it harder to read. See if you can refactor this to use a strategy pattern so that you have a different class to implement the feeding strategy for cats and for dogs. Then use a map to map breeds to feeding strategies.